index.html

```diff
@@ -35,10 +35,22 @@ <h1>DUDE, <br>Where's my Beer?</h1>
       <br>
     </form>
 
-    <button id="submit"
+    <br>
+    <button disabled id="submit"
             class="btn btn-outline">Submit</button>
     <div id="cities"></div>
     <div id="restaurants"></div>
+    <div id="results-table" class="results-table">
+      <table style="margin-bottom: 50px; width: 100%">
+        <thead>
+          <tr style="background-color: azure">
```

Comment:
We can remove styling from the HTML since it is adapted in the CSS. I had added code in the js  (lines 66-68) that makes the table header dynamic like the table contents. Did you need to change that? If yes, it looks like the styling would need to be adjusted to account for your changes. I hope this makes sense!

Reply:
yes, it added it every time I would search. You hide it then you show it.